README.md

```diff
@@ -125,7 +127,7 @@ try {
     $mail->send();
     echo 'Message has been sent';
 } catch (Exception $e) {
-    echo 'Message could not be sent. Mailer Error: ', $mail->ErrorInfo;
+    echo "Message could not be sent. Mailer Error: {$mail->ErrorInfo}";
 }
```

Comment:
The other changes are fine, but this one is not; It misunderstands how `echo` works.

Reply:
All the same.
But I think `"This is {$string}"` is more conducive to reading than `"This is", $string`, is it?

Reply:
If readability was the *only* concern, maybe, but concatenation or interpolation add unnecessary overhead (minimal though it is) with echo, so it's good practice to encourage the comma form.

Reply:
Let me do a performance benchmark, just wait a second.

Reply:
- OS: Arch x64
- PHP: 7.3.3 nts

```
<?php

// echoWithComma.php
// The result: 112bytes and 0.16124796867371s

declare(strict_types = 1);

$count         = 100000;
$isItRight     = 'is it right?';
$startMemUsage = \memory_get_usage();
$startTime     = \microtime(true);

while ($count > 0) {
    echo '<div style=display:none>Synchro said comma is good, ', $isItRight, '</div>';
    --$count;
}

$endMenUsage = \memory_get_usage() - $startMemUsage;
$endTime     = \microtime(true)    - $startTime;

echo "<h1>${endMenUsage}bytes</h1>";
echo "<h1>${endTime}s</h1>";
```
And next:
```
<?php

// echoWithoutComma.php
// The result: 112bytes and 0.10986399650574s

declare(strict_types = 1);

$count         = 100000;
$isItRight     = 'is it right?';
$startMemUsage = \memory_get_usage();
$startTime     = \microtime(true);

while ($count > 0) {
    echo "<div style=display:none>Synchro said comma is good, {$isItRight}</div>";
    --$count;
}

$endMenUsage = \memory_get_usage() - $startMemUsage;
$endTime     = \microtime(true)    - $startTime;

echo "<h1>${endMenUsage}bytes</h1>";
echo "<h1>${endTime}s</h1>";
```

Just easy to get the results. The withoutComma scheme is 68.133569% faster than withComma scheme (multi benchmark).

Reply:
You are quite right - I get similar results. So much for conventional wisdom!